packages/template-retail-react-app/package.json

```diff
@@ -89,7 +89,7 @@
         },
         {
             "path": "build/vendor.js",
-            "maxSize": "283 kB"
+            "maxSize": "310 kB"
```

Comment:
We added `@salesforce/commerce-sdk-react` as a dependency. So bumping vendor bundle size limit to allow for the added dependency.